app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,25 @@
+class VotesController < ApplicationController
+  before_filter :load_voteable
+
+  def create
+    # byebug
+    @vote = @voteable.votes.new(vote: 0,user: current_user)
+    if params[:commit] == "upvote"
+      @vote.update_attributes(vote: 1)
```

Comment:
update_attributes does an implicit save. You probably need assign_attriubtes here.